cime_config/machines/config_machines.xml

```diff
@@ -606,8 +609,10 @@
         <!-- <arg name="binding">threads-per-core=1 -c 2</arg> -->
         <arg name="binding">--threads-per-core=1</arg>
         <arg name="thread_count">-c $ENV{OMP_NUM_THREADS}</arg>
-        <arg name="placement">-m *:block</arg>
+        <arg name="distribution">-m *:block</arg>
```

Comment:
As the '*:block' deviates from the default of 'block:cyclic:cyclic', is this binding good for the GPU cases as well? Have you verified how this interacts with --gpu-bind=closest option? 

Ref for self/others: `    --distribution=<value>[:<value>][:<value>] specifies the distribution of MPI ranks across compute nodes, sockets (L3 cache regions on Crusher), and cores, respectively. `

Reply:
`block` placement (adjacent MPI tasks on adjacent cores) has always been better than `cyclic` (round-robin).
Also, all nodes and cores are almost always fully populated in E3SM runs: the only time cyclic/round-robin can run faster is when nodes or cores are under-populated.